### PR TITLE
Update PPP with latest contributions

### PR DESCRIPTION
**Pull request by ekweirui**
Update @ekweirui PPP to have the lastest contributions.

### CODE REVIEW

**Review by teoyuqi — APPROVED** [review]
LGTM :heart:

File: docs/team/ekweirui.md [hunk 1]
```diff
@@ -5,33 +5,105 @@ title: Ek Wei Rui's Project Portfolio Page
...
 * **Documentation**:
     * User Guide:
-        * Added documentation for the features `add contact` and `delete contact`
+      * Added documentation for the following features:
+        * `add-person` and `delete-person` ([#49](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/49))
+        * `edit-person` ([#62](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/62))
+        * `delete-person-from-module` ([#111](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/111))
     * Developer Guide:
-        * [coming soon]
+        * Added documentation for general contacts information and `delete-person` command
```
Comment (cheeheng) [inline docs/team/ekweirui.md:89]: Grammatical error here? To fix, perhaps you could change to this
```suggestion
        * Added documentation for general contacts-related information and `delete-person` command
```
or maybe this
```suggestion
        * Added documentation for general contact information and `delete-person` command
```

File: docs/team/ekweirui.md [hunk 2]
```diff
@@ -5,33 +5,105 @@ title: Ek Wei Rui's Project Portfolio Page
...
+          ([#88](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/88)).
 
-* **Community**:
-    * [coming soon]
 
-* **Tools**:
-    * [coming soon]
+* **Team based tasks**:
+    * PRs reviewed (with non-trivial review comments):
```
Comment (cheeheng) [inline docs/team/ekweirui.md:94]: PRs reviewed should be under separate section?
image

File: docs/team/ekweirui.md [hunk 3]
```diff
@@ -5,33 +5,105 @@ title: Ek Wei Rui's Project Portfolio Page
...
+      [#90](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/90),
+      [#101](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/101),
+      [#151](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/151),
+      [#63](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/63)
+    * Necessary general code enhancements:
+      * Refactoring of `contact` related commands to `person` after feedback from teammates
+        [#86](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/86)
+    * Maintaining the issue tracker:
+      * Created the majority of labels and milestones 
+      * Resolves issues after a relevant PR has been merged
```
Comment (cheeheng) [inline docs/team/ekweirui.md:104]: Inconsistent tense?
```suggestion
      * Resolved issues after a relevant PR has been merged
```

File: docs/team/ekweirui.md [hunk 4]
```diff
@@ -5,33 +5,105 @@ title: Ek Wei Rui's Project Portfolio Page
...
+    * PRs reviewed (with non-trivial review comments):
+      [#90](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/90),
+      [#101](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/101),
+      [#151](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/151),
+      [#63](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/63)
+    * Necessary general code enhancements:
+      * Refactoring of `contact` related commands to `person` after feedback from teammates
+        [#86](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/86)
+    * Maintaining the issue tracker:
+      * Created the majority of labels and milestones
```
Comment (cheeheng) [inline docs/team/ekweirui.md:103]: Did you mean that you put up the majority of labels and milestones to issues and pull requests? It appears that the word 'created' was misleading.

**Review by cheeheng — COMMENTED** [review]
LGTM except for some nitpicks.

**Review by cheeheng — APPROVED** [review]
Merging in because PPP draft deadline coming soon, and it will not be fair if it gets stuck in review process. Please remember that there are still feedback which needs to be addressed.

**Pull request merged**